src/Playwright/Helpers/SetInputFilesHelpers.cs

```diff
@@ -76,16 +76,16 @@ private static IEnumerable<string> GetFilesRecursive(string directory)
         return files;
     }
 
-    public static async Task<SetInputFilesFiles> ConvertInputFilesAsync(IEnumerable<string> files, BrowserContext context)
+    public static async Task<SetInputFilesFiles> ConvertInputFilesAsync(List<string> files, BrowserContext context)
```

Comment:
If they were to accept a breaking change then `IReadOnlyCollection` would allow you to pass in both and array and a list interchangeably whilst keeping the compiler happy.